docs/source/user_guide/components.ipynb

```diff
@@ -395,7 +395,7 @@
    "metadata": {},
    "source": [
     "### Hyperparameter Ranges for AutoML\n",
-    "`hyperparameter_ranges` is a dictionary mapping the parameter name (str) to an allowed range ([SkOpt Space](https://scikit-optimize.github.io/stable/modules/classes.html#module-skopt.space.space)) for that parameter.\n",
+    "`hyperparameter_ranges` is a dictionary mapping the parameter name (str) to an allowed range ([SkOpt Space](https://scikit-optimize.github.io/stable/modules/classes.html#module-skopt.space.space)) for that parameter. Both lists and `skopt.space.Categorical` values are accepted. \n",
```

Comment:
I think similar to @freddyaboulton's comment, we should clarify that only categorical spaces can be specified by either a list or skopt.space.Categorical?